### PR TITLE
add identity precompile and pallet docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR adds new pages on the identity precompile (introduced in RT2500) and the identity pallet. 

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira

### After Translation Requirements

- [x] No additional PRs are required after the translations are done

### CODE REVIEW

File: builders/pallets-precompiles/pallets/identity.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+# The Identity Pallet
+
+## Introduction {: #introduction }
+
+The [Substrate](/learn/platform/technology/#substrate-framework){target=_blank} Identity Pallet is an out-of-the-box solution for adding personal information to your on-chain account. Personal information can include default fields such as your legal name, display name, website, Twitter handle, and Riot (now known as Element) name. You can also take advantage of custom fields to include any other relevant information.
+
+The pallet also includes functionality to request judgments and verify on-chain identities from registrars, which are accounts appointed via governance to verify the identity information submitted and provide judgment on their findings for a fee.
+
+This guide will provide an overview of the extrinsics, storage methods, and getters for the pallet constants available in the Identity Pallet on Moonbeam. This guide assumes you are familiar with identity-related terminology; if not, please check out the [Managing your Account Identity](/learn/features/governance/#opengov){target=_blank} page for more information.
```
Comment (themacexpert) [inline builders/pallets-precompiles/pallets/identity.md:14]: Double check that this link is correct

File: builders/pallets-precompiles/precompiles/identity.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+- **provideJudgement**(*uint32* regIndex, *address* target, *Judgement memory* judgement, *bytes32* identity) - provides judgment on an account's identity
+
+## Interact with the Solidity Interface {: #interact-with-interface }
+
+The following sections will cover how to interact with the Identity Precompile using [Ethereum libraries](/builders/build/eth-api/libraries/){target=_blank}, such as [Ethers.js](/builders/build/eth-api/libraries/ethersjs){target=_blank}, [Web3.js](/builders/build/eth-api/libraries/web3js){target=_blank}, and [Web3.py](/builders/build/eth-api/libraries/web3py){target=_blank}.
+
+The examples in this guide will be on Moonbase Alpha.
+--8<-- 'text/common/endpoint-examples.md'
+
+### Using Ethereuem Libraries {: #use-ethereum-libraries }
```
Comment (themacexpert) [inline builders/pallets-precompiles/precompiles/identity.md:76]:
```suggestion
### Using Ethereum Libraries {: #use-ethereum-libraries }
```

File: builders/pallets-precompiles/precompiles/identity.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+[`Identity.sol`](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/identity/Identity.sol){target=_blank} is a Solidity interface that allows developers to interact with the precompile's methods.
+
+??? code "Identity.sol"
+
+    ```solidity
+    --8<-- 'code/precompiles/identity/Identity.sol'
+    ```
+
+The Identity Precompile contains some functions that can be called by anyone and some judgment-related functions that must be called by a registrar. The functions that can be called by anyone are as follows:
```
Comment (themacexpert) [inline builders/pallets-precompiles/precompiles/identity.md:46]: This is a nitpick but would "can only be called by a registrar" be more specific wording than "must be called by a registrar"?

**Review by themacexpert — APPROVED** [review]
LGTM 👍

File: builders/pallets-precompiles/precompiles/identity.md (hunk 3 above)
Comment (eshaben) [reply builders/pallets-precompiles/precompiles/identity.md:46]: Yeahh!! I'm here for it 🙂

**Comment by albertov19** [comment]
Are we waiting on anything here? This looks good to go from my perspective :)

**Pull request merged**